setup haproxy 7.1 container in konflux

sachin-yeshwanth11 opened the pull request:
Hello @andrewschoen ,

This PR contains changes required for setting up haproxy 7.1 container in Konflux.

Please help in reviewing and merging this PR, Thanks !

Commit:
setup haproxy 7.1 container in konflux

andrewschoen commented on .tekton/haproxy-7-1-pull-request.yaml:
```diff
@@ -24,12 +24,15 @@ spec:
     value: '{{revision}}'
   - name: output-image
-    value: quay.io/rhceph-dev/haproxy:on-pr-{{revision}}
+    value: quay.io/rhceph-ci/haproxy:on-pr-{{revision}}
```
can you please change the tag here to be `on-pr-v7.1-{{revision}}`

sachin-yeshwanth11 replied:
updated the tag

andrewschoen commented on .tekton/haproxy-7-1-pull-request.yaml:
```diff
@@ -24,12 +24,15 @@ spec:
   - name: build-platforms
     value:
     - linux/x86_64
+    - linux/arm64
```
arm64 can be removed as we don't build for that arch until 8.1

sachin-yeshwanth11 replied:
removed `arm64` from here

andrewschoen commented on .tekton/haproxy-7-1-push.yaml:
```diff
@@ -23,10 +23,13 @@ spec:
     value: '{{revision}}'
   - name: output-image
-    value: quay.io/rhceph-dev/haproxy:{{revision}}
+    value: quay.io/rhceph-ci/haproxy:{{revision}}
```
please change the tag here to be `v7.1-{{revision}}`

sachin-yeshwanth11 replied:
updated here as well

andrewschoen commented on .tekton/haproxy-7-1-push.yaml:
```diff
@@ -23,10 +23,13 @@ spec:
   - name: build-platforms
     value:
     - linux/x86_64
+    - linux/arm64
```
this can be removed

sachin-yeshwanth11 replied:
removed

andrewschoen reviewed (changes requested):
Great work. Same as snmp-notifier we need to remove arm64 until 8.1.

Also, can you add additional tagging like I've done here? https://github.com/ibmstorage/snmp-notifier-container/blob/release-5.3/.tekton/snmp-notifier-5-3-pull-request.yaml#L559 (side note: that v5.3 tag should be removed from the pull request pipeline)

sachin-yeshwanth11 commented:
> Great work. Same as snmp-notifier we need to remove arm64 until 8.1.
> 
> Also, can you add additional tagging like I've done here? https://github.com/ibmstorage/snmp-notifier-container/blob/release-5.3/.tekton/snmp-notifier-5-3-pull-request.yaml#L559 (side note: that v5.3 tag should be removed from the pull request pipeline)

Hello Andrew - I have addressed all the review points, requesting you to re-review. Thanks !
